ooiui/templates/science/plotting.html

```diff
@@ -362,6 +363,7 @@
         options.xvar = 'time';
       }
       
+      console.log(options.yvar)
```

Comment:
only searches in the given context/view so you wont bleed to the whole document :+1: